cdist/conf/type/__user_groups/gencode-remote

```diff
@@ -57,9 +42,25 @@ if [ -z "$changed_groups" ]; then
 fi
 
 for group in $changed_groups; do
-   if [ "$oldusermod" = "true" ]; then
-     echo "usermod $action \"$group\" \"$user\""
+   if [ "$os" = "netbsd" ]; then
```

Comment:
So many cases -- can't we just have a case for netbsd and use the old mechanism for all other OS?